ci-operator/config/openshift/cincinnati/openshift-cincinnati-release-4.8.yaml

```diff
@@ -84,17 +84,17 @@ tests:
     - as: cargo-test
       commands: env CINCINNATI_CI_DOCKERJSON_PATH="${SHARED_DIR}/${CINCINNATI_CI_DOCKERJSON_FILE}" CINCINNATI_TEST_CREDENTIALS_PATH="${SHARED_DIR}/${CINCINNATI_TEST_CREDENTIALS_FILE}" dist/cargo_test.sh
       credentials:
-      - mount_path: /usr/local/ci-secrets/cincinnati-secrets-cincinnati-credentials
-        name: cincinnati-secrets-cincinnati-credentials
+      - mount_path: /usr/local/ci-secrets/cincy-credentials
+        name: cincy-credentials
```

Comment:
[cargo-test-4.8][1]:

```
dist/cargo_test.sh: line 98: type: cargo: not found
dist/cargo_test.sh: line 98: type: docker: not found
error: could not find any of execute_native execute_docker in PATH
```

Dunno what's up with that.

/retest

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/16807/rehearse-16807-pull-ci-openshift-cincinnati-release-4.8-cargo-test/1371507519297425408#1:build-log.txt%3A32